src/coordination/include/coordination/replication_instance.hpp

```diff
@@ -18,17 +18,22 @@
 #include "replication_coordination_glue/role.hpp"
 
 #include <libnuraft/nuraft.hxx>
+#include "utils/resource_lock.hpp"
 #include "utils/result.hpp"
 #include "utils/uuid.hpp"
 
 namespace memgraph::coordination {
 
 class CoordinatorInstance;
+class ReplicationInstance;
+
+using HealthCheckInstanceCallback = void (CoordinatorInstance::*)(ReplicationInstance &);
```

Comment:
What was wrong with previous definition?

Reply:
We were sending `instance_name` and then doing find instance, so I don't see why we should do that 2 times

Reply:
Reverted back